src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -117,7 +117,7 @@
      * - the {@code actualModel} matches {@code expectedModel}
      */
     public static void assertCommandSuccess(Command command, Model actualModel, CommandResult expectedCommandResult,
-            Model expectedModel) {
+                                            Model expectedModel) {
```

Comment:
didn't know this passes checkstyle 😮

Reply:
Haha configured my IntelliJ to FormatOnSave according to our checkstyle standards so there's some random changes like this